lib/include/ultrahdr/jpegr.h

```diff
@@ -27,7 +27,7 @@
 namespace ultrahdr {
 
 // The current JPEGR version that we encode to
-static const char* const kJpegrVersion = "1.0";
+static const char* const kJpegrVersion = kGainMapVersion;
```

Comment:
nit; can this change be used to correct all references to "1.0" to kJpegRVersion
appendGainMap routine has 
`  if (metadata->version.compare("1.0")) {
    ALOGE("received bad value for version: %s", metadata->version.c_str());
    return ERROR_JPEGR_BAD_METADATA;
  }`
jpegr_test.cpp has
`  good_metadata.version = "1.0";` and
`  metadata_expected.version = "1.0";`

Further if we intend to remove kJpegrVersion completely in the future (once aosp changes are made) because it seems to be an alias to kGainMapVersion.

Reply:
we can add a todo

Reply:
I think we can keep this as is for now since the format is not finalized yet. I defined `kGainMapVersion` and `kJpegrVersion` and they may be different in the future. e.g. `kGainMapVersion` can be "2.0" and iso metadata will use 2.0 version, while `kJpegrVersion` can still be "1.0" and XMP uses 1.0 metadata to keep the backward compatibility.